templates/common/_base/files/configure-ovs-network.yaml

```diff
@@ -177,25 +204,38 @@ contents:
         iface_type=802-3-ethernet
       fi
 
-      # bring down any old iface
-      nmcli device disconnect $iface
-
       # use ${extra_phys_args[@]+"${extra_phys_args[@]}"} instead of ${extra_phys_args[@]} to be compatible with bash 4.2 in RHEL7.9
       extra_args=${extra_phys_args[@]+"${extra_phys_args[@]}"}
```

Comment:
shellcheck warning
```
configure-ovs-network.sh:204:14: warning: Assigning an array to a string! Assign as array, or use * instead of @ to concatenate. [SC2124]
```